utils/eventtracking.js

```diff
@@ -40,41 +47,49 @@ const track = async (eventType, eventProperties, options) => {
     }
     try {
         const mixPanelProperties = {
-            distinct_id: isGitPod() ? getGitPodUserHash() : shellSettings[TRACKING_SESSION_ID_KEY],
+            distinct_id: isGitPod()
+                ? getGitPodUserHash()
+                : shellSettings[TRACKING_SESSION_ID_KEY],
             near_cli_version,
             os: process.platform,
             network_id: options.networkId,
             node_url: options.nodeUrl,
             wallet_url: options.walletUrl,
-            is_gitpod: isGitPod()
+            is_gitpod: isGitPod(),
         };
         Object.assign(mixPanelProperties, eventProperties);
         await mixpanel.track(eventType, mixPanelProperties);
     } catch (e) {
-        console.log('Warning: problem while sending developer event tracking data. This is not critical. Error: ', e);
+        console.log(
+            'Warning: problem while sending developer event tracking data. This is not critical. Error: ',
+            e
+        );
     }
 };
 
 const getEventTrackingConsent = async () => {
     const rl = readline.createInterface({
         input: process.stdin,
-        output: process.stdout
+        output: process.stdout,
     });
     try {
         for (let attempts = 0; attempts < 10; attempts++) {
             const answer = await new Promise((resolve) => {
                 rl.question(
                     chalk`We would like to collect data on near-cli usage to improve developer experience.` +
-                    chalk` We will never send private information. We only collect which commands are run via an anonymous identifier.` +
-                    chalk`{bold.yellow  Would you like to opt in (y/n)? }`,
+                        chalk` We will never send private information. We only collect which commands are run via an anonymous identifier.` +
+                        chalk`{bold.yellow  Would you like to opt in (y/n)? }`,
                     async (consentToEventTracking) => {
-                        if (consentToEventTracking == 'y' || consentToEventTracking == 'Y') {
+                        if (consentToEventTracking.toLowerCase() == 'y') {
                             resolve(true);
-                        } else if (consentToEventTracking == 'n' || consentToEventTracking == 'N') {
+                        } else if (
+                            consentToEventTracking.toLowerCase() == 'n'
```

Comment:
nit: formatting is a bit weird (no reason not to keep `if` and condition on same line)